### DISCUSSION TITLE
Profile picture upload fails when using S3

### DISCUSSION THREAD

**Discussion by AutonomousWork** [outline/outline#7519] [Self Hosting]
Profile Image uploads fail when s3 is set for file storage. No issues uploading images within documents.

**To Reproduce**
1. Click profile or organization icon in settings
2. Attempt to upload picture
3. Upload fails

**Expected behavior**
Upload successful 

**Screenshots**
If applicable, add screenshots or videos to help explain your problem.

**Outline (please complete the following information):**
- Install: self hosted
- Version: 0.78.0

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro
 - Browser: Edge

**Mobile (please complete the following information):**
 - Device: iPhone 14 Pro Max
 - OS: 17.6.1
 - Browser: Safari

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-604/profile-picture-upload-fails-when-using-s3">OLN-604 Profile picture upload fails when using S3</a></p>
